src/main/java/seedu/address/commons/core/Messages.java

```diff
@@ -13,7 +13,7 @@
     public static final String MESSAGE_PERSON_NOT_FOUND = "The person '%1$s' is not registered";
     public static final String MESSAGE_INVALID_ROOM = "the room does not exist";
     public static final String MESSAGE_INVAILD_REFERENCE_ID =
-            "The reference ID '%1$s' does not belong to any registered person!";
+            "The referenceId '%1$s' does not belong to any registered person!";
```

Comment:
As mentioned in pull request #165. For command results, "referenceId" should be 2 words. So "reference Id"

Reply:
okay